From ee57303cf0eb3640798d4c25b49ec13f5ddcbad3 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Thu, 8 Jul 2010 10:21:11 +0800
Subject: [PATCH 167/449] ENGR00124196 vpu: Change vpu unit test to sync with
 v1.4.0 firmware

Change vpu test to sync with v1.4.0 firmware on mx51 and mx53;
Remove underrun processing since the feature is dropped from
firmware.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 test/mxc_vpu_test/dec.c | 6 ------
 test/mxc_vpu_test/enc.c | 6 ------
 2 files changed, 12 deletions(-)

diff --git a/test/mxc_vpu_test/dec.c b/test/mxc_vpu_test/dec.c
index 9410c84..8cb4de5 100644
--- a/test/mxc_vpu_test/dec.c
+++ b/test/mxc_vpu_test/dec.c
@@ -807,12 +807,6 @@ decoder_start(struct decode *dec)
 			vpu_DecGiveCommand(handle, SET_DBK_OFFSET, &dbkOffset);
 		}
 
-		/* RTC setting */
-		decparam.underrunEnable = 0;
-		decparam.accumulativeMode = 0;
-		decparam.pulseWidth = 0;
-		decparam.refMbClk = 0;
-
 		gettimeofday(&tdec_begin, NULL);
 		ret = vpu_DecStartOneFrame(handle, &decparam);
 		if (ret != RETCODE_SUCCESS) {
diff --git a/test/mxc_vpu_test/enc.c b/test/mxc_vpu_test/enc.c
index 4b2828d..02cc249 100644
--- a/test/mxc_vpu_test/enc.c
+++ b/test/mxc_vpu_test/enc.c
@@ -456,12 +456,6 @@ encoder_start(struct encode *enc)
 				break;
 		}
 
-		/* RTC setting */
-		enc_param.underrunEnable = 0;
-		enc_param.accumulativeMode = 0;
-		enc_param.pulseWidth = 0;
-		enc_param.refMbClk = 0;
-
 		gettimeofday(&tenc_begin, NULL);
 		ret = vpu_EncStartOneFrame(handle, &enc_param);
 		if (ret != RETCODE_SUCCESS) {
-- 
1.8.0

